knowledge_base/bundle_variables/README.md

```diff
@@ -0,0 +1,65 @@
+# bundle_variables
+
+Example to show how bundle variables work.
+
+We will focus on the behavior of the variables section alone. To do so, we exclusively use
+the `validate` command, and use its JSON output mode to focus on the variables section.
+
+## Usage
+
+Configure the workspace to use:
+```sh
+export DATABRICKS_CONFIG_PROFILE="<workspace profile>"
+```
+
+Try to run validate without arguments:
+```sh
+databricks bundle validate --output json | jq .variables
+```
+
+Because the configuration includes a variable definition without a value, it returns an error
+saying that the value must be defined:
+```
+Error: no value assigned to required variable "no_default".
+```
+
+Assign a value for this variable by either:
+
+1. Adding a `--var` flag to all bundle commands:
```

Comment:
```suggestion
- Add a `--var` flag to all bundle commands:
```

Reply:
These aren't steps (numbered list) but a list of options (bulleted list).  However, I like making it really clear (see the next comment)...